diff --git a/node_modules/@solana/web3.js/lib/index.browser.esm.js b/node_modules/@solana/web3.js/lib/index.browser.esm.js
index 8b39556..5c49e6c 100644
--- a/node_modules/@solana/web3.js/lib/index.browser.esm.js
+++ b/node_modules/@solana/web3.js/lib/index.browser.esm.js
@@ -2622,7 +2622,7 @@ function createRpcClient(url, useHttps, httpHeaders, fetchMiddleware, disableRet
       return new Promise((resolve, reject) => {
         fetchMiddleware(url, options, async (url, options) => {
           try {
-            resolve(await fetch(url, options));
+            resolve(await globalThis.fetch(url, options));
           } catch (error) {
             reject(error);
           }
@@ -2651,7 +2651,7 @@ function createRpcClient(url, useHttps, httpHeaders, fetchMiddleware, disableRet
         if (fetchWithMiddleware) {
           res = await fetchWithMiddleware(url, options);
         } else {
-          res = await fetch(url, options);
+          res = await globalThis.fetch(url, options);
         }
 
         if (res.status !== 429
