diff --git a/node_modules/node-fetch/browser.js b/node_modules/node-fetch/browser.js
index 83c54c5..2008fe8 100644
--- a/node_modules/node-fetch/browser.js
+++ b/node_modules/node-fetch/browser.js
@@ -16,9 +16,9 @@ var global = getGlobal();
 module.exports = exports = global.fetch;
 
 // Needed for TypeScript and Webpack.
-if (global.fetch) {
-	exports.default = global.fetch.bind(global);
-}
+// if (global.fetch) {
+// 	exports.default = global.fetch.bind(global);
+// }
 
 exports.Headers = global.Headers;
 exports.Request = global.Request;
